data/campaigns/Eastern_Invasion/scenarios/07a_The_Crossing.cfg

```diff
@@ -31,6 +31,7 @@
         id=Garnad
         name= _ "Garnad"
         side=2
+        hidden=yes # until it appears
```

Comment:
I'm going to drop this - if the player opens the status dialog, it seems better to me to show it rather than show sides 1, 3 and 4.